app/Models/Bookmark.php

```diff
@@ -36,14 +45,54 @@ public function bookmarkable()
 
     #endregion
 
-    #region Scopes
-
-    public function scopeWhereBills(Builder $query)
+    public function scopePerGroup(Builder $query, Group $group)
```

Comment:
Running `composer cs-fix` should fix all of the whitespace formatting issues in the following ~50 lines